doc/api/net.md

```diff
@@ -625,15 +625,16 @@ initialDelay will leave the value unchanged from the default
 
 Returns `socket`.
 
-### socket.setNoDelay([noDelay])
+### socket.setNoDelay([enable])
```

Comment:
That said, I'm not sure if it really makes sense to rename the argument to `enable`.

Thoughs?